modules/apps/journal/journal-web/src/main/resources/META-INF/resources/js/JournalPortlet.es.js

```diff
@@ -279,7 +279,27 @@ class JournalPortlet extends PortletBase {
 		this._cleanInputIfNeeded('titleMapAsXML');
 		this._cleanInputIfNeeded('descriptionMapAsXML');
 
+		const submitButtons = Array.from(
+			form.querySelectorAll(`${BUTTON_ROW_CLASS} button[type=submit]`)
+		);
+
+		submitButtons.forEach((button) => {
+			button.disabled = true;
+		});
+
 		submitForm(form);
```

Comment:
Hey @victorg1991, `submitForm` can be called with [4 arguments](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/events.js#L63) `(form, action, singleSubmit, validate)`. 

The third one, `singleSubmit` [should prevent further submissions from happening](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/events.js#L113-L125).

Could you check to see if that fixes this issue so we can leverage it (or fix it if needed) instead of rolling out our own logic here?